### DISCUSSION TITLE
how can i display text at the corner of the axis

### DISCUSSION THREAD

**Discussion by hongpatipan** [apexcharts/apexcharts.js#4722] [General]
how can i display text at the corner of the axis

![image](https://github.com/user-attachments/assets/baeb3303-6ca0-4e79-aa42-215906aa034a)

**Comment by hongpatipan** [comment]
Please answer me

**Comment by misakstvanu** [comment]
Where is this screenshot from?

**Reply by hongpatipan** [reply]
APEXCHART HEAT MAP
